### PR TITLE
Fingerprint RSpec

### PR DESCRIPTION
**Pull request by CamAnNguyen**
Update fingerprint RSpec

### CODE REVIEW

File: spec/models/fingerprint_spec.rb [hunk 1]
```diff
@@ -62,4 +62,56 @@
...
+
+      expect(standard.include? 'R#').to be(false)
+      expect(standard.include? 'PolymersList').to be(false)
+    end
+
+    it 'should create a fingerprint' do
+      fp_id = Fingerprint.find_or_create_by_molfile(polymer_molfile)
+      fp = Fingerprint.find(fp_id)
+
+      expect(fp.fp0).to eq("0000000000000000000000000000000000000000000000000000000000000000")
```
Comment (Sergeykot) [inline spec/models/fingerprint_spec.rb]: when the string contains only from "0" I would user "0" * n
n is an Integer

**Comment by Sergeykot** [comment]
plz squash in one commit

**Comment by Sergeykot** [comment]
git rebase -i HEAD~2

File: spec/models/fingerprint_spec.rb (hunk 1 above)
Comment (CamAnNguyen) [inline spec/models/fingerprint_spec.rb]: It's more simple for me when I manually generated a 64-bits fingerprint. It's also easy to clearly see the fingerprint I want to test, that the generated fingerprint should be exactly like this. Can I just keep it like this?

Comment (Sergeykot) [inline spec/models/fingerprint_spec.rb]: yes sure, now I agree with you

**Comment by Sergeykot** [comment]
:+1:

**Pull request merged**